source/assets/sass/vendor/_syntax-accessible.scss

```diff
@@ -1,273 +1,110 @@
 @use '../functions';
 
-// Stylesheet pulled from Rogue stylesheet with colours updated for a11y
+/**
+ * Based on the GHColors theme by Avi Aryan (http://aviaryan.in)
+ * Inspired by GitHub syntax coloring
+ */
 
-.highlight table td { padding: functions.sl-px-to-rem(5px); }
-.highlight table pre { margin: 0; }
-.highlight .cm {
-  color: #656556;
-  font-style: italic;
-}
-
-.highlight .cp {
-  color: #999999;
-  font-weight: bold;
-}
-
-.highlight .c1 {
-  color: #656556;
-  font-style: italic;
-}
-
-.highlight .cs {
-  color: #999999;
-  font-weight: bold;
-  font-style: italic;
-}
-
-.highlight .c, .highlight .cd {
-  color: #006666;
-  font-style: italic;
-}
-
-.highlight .err {
-  color: #a61717;
-  background-color: #e3d2d2;
-}
-
-.highlight .gd {
-  color: #000000;
-  background-color: #ffdddd;
-}
-
-.highlight .ge {
-  color: #000000;
-  font-style: italic;
-}
-
-.highlight .gr {
-  color: #aa0000;
-}
-
-.highlight .gh {
-  color: #999999;
-}
-
-.highlight .gi {
-  color: #000000;
-  background-color: #ddffdd;
+code[class*="language-"],
+pre[class*="language-"] {
+  background-color: var(--sl-color-code--bg);
+  hyphens: none;
+  word-break: normal;
+	word-spacing: normal;
+	tab-size: 4;
```

Comment:
> Is the indent here intentional? I know this is a vendor file, but I'll add this one to the CSS clean up Trello card checklist.

Hmm, it seems like that's the tab-size they use currently. But this could probably be removed since it's also set in their reset file.